docs: update Python migration guide for drop-callback change (#81)
atolab/zenoh-web: content/docs/migration_1.0/Python.md

    # and it will undeclare the subscriber
```

## Drop-callback has to be wrapped in `handlers.Callback`

> In previous version, it was possible to pass a drop-callback with the main callback in a tuple for operations like `Session.declare_subscriber`. However, it was also possible to pass a tuple with a "receiver" (renamed "handler" in 1.0) as second member, and that could be confusing

Reviewer: ```suggestion
In the previous 0.11.0 version, it was possible to pass a drop-callback with the main callback in a tuple for operations like `Session.declare_subscriber`. However, it was also possible to pass a tuple with a "receiver" (renamed "handler" in 1.0.0) as second member, and that could lead to confuse for users.
```

> In the previous 0.11.0 version, it was possible to pass a drop-callback with the main callback in a tuple for operations like `Session.declare_subscriber`. However, it was also possible to pass a tuple with a "receiver" (renamed "handler" in 1.0.0) as second member, and that could lead to confuse for users.

Reviewer 2: ```suggestion
In the previous 0.11.0 version, it was possible to pass a drop-callback with the main callback in a tuple for operations like `Session.declare_subscriber`. However, it was also possible to pass a tuple with a "receiver" (renamed "handler" in 1.0.0) as second member, and that could lead to confusion for users.
```

Reviewer 3: > and that could confuse users.

The audience is the user and so the user should be addressed directly.

Reviewer 2: My suggestion was follow-up to Luca's suggestion which wasn't applied correctly in the previous commit.

<br>
The API has been changed and now requires the drop-callback to be wrapped in `handlers.Callback`. 

- Zenoh 0.11.x

```python
def on_sample(sample: zenoh.Sample): ...
def on_done(): ...
session.declare_subscriber((on_sample, on_done))
```

- Zenoh 1.0.0

```python
def on_sample(sample: zenoh.Sample): ...
def on_done(): ...
session.declare_subscriber(zenoh.handlers.Callback(on_sample, on_done))
```

> NOTE: ⚠️ Passing drop-callback in a tuple will no more work as expected, as the drop callback will never be executed. To ease migration and avoid surprises, a warning will be displayed in this case.

Reviewer 2: ```suggestion
NOTE: ⚠️ Passing drop-callback in a tuple will no longer work as expected, as the drop callback will never be executed. To ease migration and avoid surprises, a warning will be displayed in this case.
```
